README.md

```diff
@@ -205,6 +205,9 @@ export default withRelay(UserProfile, UserProfileQuery, {
   // Create a Relay environment on the client-side.
   // Note: This function must always return the same value.
   createClientEnvironment: () => getClientEnvironment()!,
+  // variablesFromContext allows you to declare and customize variables for the graphql query.
+  // by default variablesFromContext is ctx.query
+  variablesFromContext: (ctx: NextRouter | NextPageContext) => ({ ...ctx.query, otherVariable: true }),
```

Comment:
could import the types, but I feel like if you've made it this far you know where they live 🤷‍♂️

Reply:
Yea this is fine, should show up as a recommended import anyway